oemof/solph/inputlib/csv_tools.py

```diff
@@ -11,7 +11,7 @@
 
 def NodesFromCSV(file_nodes_flows, file_nodes_flows_sequences,
                  delimiter=',', additional_classes={},
-                 additional_seq_attributes=[], addtional_flow_attributes=[]):
+                 additional_seq_attributes=[], additional_flow_attributes=[]):
```

Comment:
I think we should. Just change the default argument to None (`additional_classes=None`) and add an if-clause.

```python
if additional_classes is None:
    additional_classes = dict() #  or list()
```